sc_cost_meter/app.py

```diff
@@ -1,42 +1,30 @@
-import json
-
-# import requests
+import logging
+import sc_cost_meter.utils as utils
+
+log = logging.getLogger(__name__)
+log.setLevel(logging.DEBUG)
+
+
+def meter_instances():
+  '''
+  Send EC2 cost info to the AWS Marketplace
+  '''
+  instances = utils.get_instances(['running'])
```

Comment:
What about buckets (storage costs, egress costs, etc.)?  Can we bill for that too?